src/android/CodePushPackageManager.java

```diff
@@ -95,4 +95,16 @@ public InstallOptions getPendingInstall() {
     public void clearPendingInstall() {
         this.codePushPreferences.clearPendingInstall();
     }
+
+    public void markUnconfirmedInstall(boolean unconfirmed) {
```

Comment:
The implementation was done that way because we only care about two states (is confirmed / is not confirmed), and from the `CodePushPackageManager` perspective we only see two states. That is the reason for which I consider a single method with a boolean parameter would do it. Internally we have 3 states (true / false / not set), with the last two states being the same. As long as we are state-consistent in the internal implementation, we should be fine, and we do have consistency.